src/libraries/Microsoft.Extensions.Hosting/tests/UnitTests/Microsoft.Extensions.Hosting.Unit.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent);net461</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-Windows;$(NetCoreAppCurrent);net461</TargetFrameworks>
```

Comment:
OS-specific TFMs in regular SDK are all lower-case, e.g. https://docs.microsoft.com/en-us/dotnet/standard/frameworks#net-5-os-specific-tfms . Should we match it?

Reply:
We are using uppercase for all the other platforms like Android, ivOS, Unix. We will need then change all that as well to uppercase.

Reply:
why not do it now for windows so that we won't need to change it later?